conda_forge_tick/make_graph.py

```diff
@@ -173,6 +174,11 @@ def make_graph(
 
     logger.info("inferring nodes and edges")
 
+    # pluck anything marked as archived
+    for name in list(gx.nodes):
+        if gx.nodes[name].get("payload", {}).get("archived", False):
+            pluck(gx, name)
```

Comment:
Are we sure we want to do that here? If it is a true dep we should have it in the graph. For instance if a migration needs to run through an archived feedstock then that is something we'll need to handle, either by moving the dep or by reviving the feedstock.